### PR TITLE
test (gradle-plugin/it) : Add integration test for ProjectLabelEnricher's app configuration option (#1262)

### LINKED ISSUE #1262: Gradle-Enricher: ProjectLabelEnricher
### Description

Child of #31

ProjectLabelEnricher works with Gradle

### AC
- [x] Enricher documentation is ported to Gradle #1331
- [x] Create IT to verify that `jkube.enricher.jkube-project-label.app` flag works as expected #1386
- [x] kubernetes.yml is generated as expected #1331
- [x] openshift.yml is generated as expected #1331

### PR DESCRIPTION
**Pull request by rohanKanojia**
Fix #1262

Follow up to #1331

Add an integration test to verify `jkube.enricher.jkube-project-label.app`
flag works as expected

Signed-off-by: Rohan Kumar <rohaan@redhat.com>

## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [X] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [X] I have read the [contributing guidelines](https://www.eclipse.org/jkube/contributing)
 - [X] I signed-off my commit with a user that has signed the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php)
 - [ ] I Added [CHANGELOG](../CHANGELOG.md) entry
 - [ ] I have implemented unit tests to cover my changes
 - [ ] I have updated the [documentation](../kubernetes-maven-plugin/doc) accordingly
 - [ ] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=jkubeio_jkube) report
 - [ ] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

### CODE REVIEW

File: gradle-plugin/it/src/test/java/org/eclipse/jkube/gradle/plugin/tests/ProjectLabelAppIT.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+public class ProjectLabelAppIT {
+  @Rule
+  public final ITGradleRunner gradleRunner = new ITGradleRunner();
+
+  @Test
+  public void k8sResource_whenRun_generatesK8sManifestsWithAppLabels() throws IOException, ParseException {
+    // When
+    final BuildResult result = gradleRunner.withITProject("project-label")
+        .withArguments("k8sResource", "--stacktrace", "-Pjkube.enricher.jkube-project-label.app=custom-app")
```
Comment (sunix) [inline gradle-plugin/it/src/test/java/org/eclipse/jkube/gradle/plugin/tests/ProjectLabelAppIT.java:34]: what is `-P` for ?

Comment (sunix) [reply gradle-plugin/it/src/test/java/org/eclipse/jkube/gradle/plugin/tests/ProjectLabelAppIT.java:34]: is it similar to `-D`?

**Comment by rohanKanojia** [comment]
oh, I'm not sure what I was thinking :confused: . Let me add another entry

**Pull request merged**